spec/board_spec.rb

```diff
@@ -0,0 +1,54 @@
+require './lib/board'
+
+RSpec.describe Board do
+  let(:empty_arr) { ['', '', '', '', '', '', '', '', ''] }
+  let(:board) { Board.new }
+
+  context 'Create Board instances' do
+    it 'make sure that at the beginning we get an empty array' do
+      expect(Board.new.board).to eql(empty_arr)
+    end
+  end
+
+  context '#add_marker' do
+    it 'raise an ArgumentError if we tried to add a marker in the same index multiple times' do
+      board.add_marker('X', 1)
+      expect { board.add_marker('X', 1) }.to raise_error(ArgumentError, 'Error: Square already filled, try again')
+    end
+
+    it 'change the board array by adding a new symbol to it' do
+      board.add_marker('X', 1)
+      expect(board.board).not_to eql(empty_arr)
+    end
+  end
+
+  context '#winner' do
```

Comment:
* [ ] Specs for each method should be wrapped in one `describe` block. As in example here https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples . ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/describe.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))

- change the context on line #25 to be a  `describe` block.